src/content/ui-api/observability/prometheus/disable-monitoring/index.md

```diff
@@ -0,0 +1,80 @@
+---
+linkTitle: Disable Monitoring
+title: Disable Monitoring
+description: A guide explaining how to disable monitoring for a cluster
+weight: 50
+menu:
+  main:
+    identifier: uiapi-observability-clustermonitoring
+    parent: uiapi-observability-prometheus
+user_questions:
+  - How can I disable monitoring on my cluster ?
+aliases:
+  - /observability/prometheus/disable-monitoring
```

Comment:
Aliases are only needed in case you are moving a page to a different URL. They make sure that requests will be redirected.

```suggestion
```